distribution/docker/src/docker/Dockerfile

```diff
@@ -117,27 +115,6 @@ RUN sed -i -e 's/ES_DISTRIBUTION_TYPE=tar/ES_DISTRIBUTION_TYPE=docker/' bin/elas
     chmod 0775 bin config config/jvm.options.d data logs plugins && \\
     find config -type f -exec chmod 0664 {} +
 
-<% if (docker_base == "cloud") { %>
```

Comment:
I've removed the blocks for `cloud` from Dockerfile template, because I believe it is no longer used. I couldn't see any code which would use that. Please let me know if I am missing something here.

Reply:
yes it was removed here: https://github.com/elastic/elasticsearch/pull/115357